dataland-email-service/DockerfileBase

```diff
@@ -0,0 +1,9 @@
+ARG DATALAND_GRADLE_BASE_VERSION
+FROM ghcr.io/d-fine/dataland/dataland_gradle_base:${DATALAND_GRADLE_BASE_VERSION} as builder
+
+WORKDIR /app
+COPY ./dataland-email-service/ ./dataland-email-service/
+COPY ./dataland-backend-utils/ ./dataland-backend-utils/
+COPY ./dataland-message-queue-utils/ ./dataland-message-queue-utils/
+COPY ./.git/ ./.git
```

Comment:
Warum kopieren wir eigentlich den `.git?` folder ins image?

Reply:
Das weiß ich tatsächlich auch nicht. Aber den hat es wegen dem git plugin gebraucht. Und das gitplugin war aus irgendeinem anderen grund notwendig.

Reply:
Hat das was damit zu tun, dass man hinter "/gitinfo" die Infos über den commit und branch-Namen sehen soll in deployments?

Falls ja, dann würde das aber ausreichen, wenn das nur im backend-container passieren würde.

In allen anderen images könnte man diesen git-Ordner dann vermutlich rausschmeißen.

Sollen wirs testen?

Reply:
War für den build nicht notwendig. Das git gradle plugin habe ich rausgeworfen im email service. Wir wollen noch ermitteln, zu welchem Zweck der .git folder auf allen services kopiert wird.